RadarSDK/Include/Radar.h

```diff
@@ -805,7 +805,7 @@ logConversionWithNotification
           completionHandler:(RadarSearchGeofencesCompletionHandler)completionHandler NS_SWIFT_NAME(searchGeofences(near:radius:tags:metadata:limit:completionHandler:));
 
 /**
- Autocompletes partial addresses and place names, sorted by relevance.
+ Autocompletes partial addresses and place names, sorted by relevance. (Deprecated)
```

Comment:
Hmmm, let's say something more than just "deprecated", as that might lead someone to believe that the entire function is dep'd. Perhaps "(Deprecated — prefer newer function defined above)"

Reply:
Or something like that — maybe you can come up with even more clear, succinct phrasing